lerna-package-lock.json

```diff
@@ -34567,7 +34566,7 @@
 			"integrity": "sha512-OapHwT5Ug1c3iiv8JNrwh/3XGjP45l/2iwfl25chePrvCgl0786ZvsJK+hxCCaUZjPm1lHdh5LHZhmKRxunziA=="
 		},
 		"fluid-framework-previous": {
-			"version": "npm:fluid-framework-previous@2.0.0-internal.2.0.0",
+			"version": "npm:fluid-framework@2.0.0-internal.2.0.0",
```

Comment:
Whats with all these changes?

Reply:
I haven't found a cause for this, but as it repros for multiple developers and the hash does not change, its probably better to merge this.